<launch>

	<arg name="quadrotor"/>
	<arg name="bag"/>

	<node ns="$(arg quadrotor)" name="reactive_planner" pkg="acl_planning" type="reactive_planner" output="screen" clear_params="true">

		<rosparam file="$(find acl_planning)/param/react.yaml" command="load"/>

		<remap from="~vicon" to="state_estimate"/>
		<remap from="~goal" to="goal" />
		<remap from="~event" to="event" />
		<remap from="~global_goal" to="global_goal" />


		<remap from="~new_global_goal" to="new_global_goal"/>
		<remap from="~int_goal" to="int_goal"/>	
		<remap from="~last_global_goal" to="last_global_goal"/>
		<remap from="~traj" to="traj"/>
		<remap from="~latency" to="latency"/>
		<remap from="~camera/cloud" to="camera/cloud"/>	


	</node>

	<group if="$(arg bag)">
        <!-- Ensure RosBag is saving important data -->
        <node ns="$(arg quadrotor)" pkg="rosbag" type="record" name="rosbag_record_planner" args="-o /media/odroid/LOGGER_USB1/bags/$(arg quadrotor)PlannerLogFile global_goal int_goal new_global_goal last_global_goal traj event latency /tf"/>
    </group>

</launch>